src/legacy/shared/Banner/Banner.tsx

```diff
@@ -39,18 +37,18 @@ const Banner = (props: Props) => {
           <span>{props.message}</span>
         }
       </div>
-      <AppConsumer>
-        {({ screenType }: AppConsumerProps) => {
-          return (
-            <CloseButton
-              className="close"
-              height={screenType < ScreenType.TABLET ? '32px' : '56px'}
-              iconColor="upper"
-              onClose={props.onClose}
-              width={screenType < ScreenType.TABLET ? '32px' : '56px'} />
-          );
-        }}
-      </AppConsumer>
+      <CloseButton
+        className="d-flex d-md-none close"
+        height="32px"
+        iconColor="upper"
+        onClose={props.onClose}
+        width="32px" />
+      <CloseButton
```

Comment:
Is this a copy pasta error? i see two close buttons

Reply:
When I removed the `AppConsumer` we didn't have a way to check if the user was on `mobile`, `tablet`, or `desktop` so I used two components and used `display: none` depending on the screen size (given by bootstrap)

Reply:
ah i see; can we assign the `32px` to a variable like `mobileButtonSize` to make it more apparent you are doing this?

Reply:
Sounds good!